### DISCUSSION TITLE
iCon spiegeln nur über CSS möglich?

### DISCUSSION THREAD

**Discussion by mcuiobroker** [Zefau/ioBroker.jarvis#1993] [Q&A]
Um ein Icon zu spiegeln nutzt man transform: scale(-1,1). Dies wird aber nur über CSS übernommen, nicht in dem Icon JSON.

`{"default":{"transform":"scale(-1,1)"}}`

Es erscheint danach kein Icon mehr. Warum?

In CSS funktioniert es. Es wird gespiegelt.
.jarvis-widget-15129047-0c87-4a6c-b2d0-c57ab7d35a17 .jarvis-StateListItem-Icon  svg {
	transform: scale(-1,1);
}

- [ ] translate() funktioniert gar nicht, weder im Icon-JSON noch im CSS?

**Comment by Zefau** [comment]
Geht 
`{"default":{"transform":"scale('-1,1')"}}`
oder
`{"default":{"transform":"scale('-1','1')"}}` ?

**Reply by Zefau** [reply]
bzw. geht es mit Reload?

**Reply by mcuiobroker** [reply]
Nein, beide Version funktionieren nicht. Auch mit Reload nicht. Icon verschwindet.

**Reply by mcuiobroker** [reply]
Fehler gefunden, hatte es nicht im Stil drin, sondern im Icon. translate muss ich noch prüfen.

**Comment by mcuiobroker** [comment]
Nutzt man zusätzlich translate  erscheint wieder das ursprüngliche Icon.
{
  "1200": {
    "color": "yellow",
    "transform": "scale(-1,1) translate(0 -5)"
  }
}

Hat keine Priorität!

**Comment by Zefau** [comment]
Und `transform: rotate(-180deg)` ?

**Reply by mcuiobroker** [reply]
Macht er mit:
![image](https://user-images.githubusercontent.com/84818934/197515615-8145985b-d142-4f35-a555-f7ade018fcf1.png)
